README.md

```diff
@@ -358,6 +358,23 @@ Templates in React are written inside the JavaScript file using the [JSX languag
 </PrimaryButton>
 ```
 
+### VueJs
```

Comment:
Could you write it as `Vue.js` ? I think that's the official name.

Reply:
Sure, will update all my outstanding PRs to use Vue.js